_includes/payment-resource.md

```diff
@@ -197,7 +197,8 @@ Content-Type: application/json
 | └➔&nbsp;`prices`         | `object`     | The `prices` resource lists the prices related to a specific payment.                                                                                                                                                                                                                                                                                      |
 | └➔&nbsp;`prices.id`      | `string`     | {% include field-description-id.md resource="prices" %}                                                                                                                                                                                                                                                                                                    |
 | └➔&nbsp;`description`    | `string(40)` | {% include field-description-description.md documentation_section=documentation_section %}                                                                                                                                                                                                                                                                 |
-| └➔&nbsp;`payerReference` | `string`     | The reference to the payer (consumer/end-user) from the merchant system, like e-mail address, mobile number, customer number etc.                                                                                                                                                                                                                          |
+| └➔&nbsp;`payerReference` | `string`     | The reference to the payer (consumer/end user) from the merchant system. Can be mobile number, e-mail address, account number, a UUID, etc.                                                                                                    
```

Comment:
Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)